### PR TITLE
feat(native): add user feedback feature description

### PR DESCRIPTION
**Pull request by tustanivsky**
This PR adds a user feedback feature description for `sentry-native` which was introduced in https://github.com/getsentry/sentry-native/pull/966

### CODE REVIEW

**Review by kahest — APPROVED** [review]
LGTM, thank you @tustanivsky!

File: docs/platforms/native/user-feedback/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Set Up User Feedback
+description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
```
Comment (lizokm) [inline docs/platforms/native/user-feedback/index.mdx:3]:
```suggestion
description: "Learn how to view user feedback submissions which, paired with the original event, give you additional insight into issues."
```

File: docs/platforms/native/user-feedback/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+sidebar_order: 6000
+---
+
+When a user experiences an error, Sentry provides the ability to collect additional feedback. You can collect feedback according to the method supported by the SDK.
```
Comment (lizokm) [inline docs/platforms/native/user-feedback/index.mdx:7]:
```suggestion
Sentry makes it possible to collect additional feedback when a user experiences an error. You can collect feedback according to the method supported by your SDK of choice.
```

File: docs/platforms/native/user-feedback/index.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## User Feedback API
+
+The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.
+
+Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. For example, to get the `eventId`, you can use <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> or the return value of the method capturing an event.
+
+<PlatformContent includePath="user-feedback/sdk-api-example/" />
+
+Alternatively, you can use the [User Feedback API endpoint](/api/projects/submit-user-feedback/) directly.
```
Comment (lizokm) [inline docs/platforms/native/user-feedback/index.mdx:17]:
```suggestion
The user feedback API allows you to collect user feedback using your own UI. You can use the same programming language you have in your app to then send the feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.

Sentry pairs the feedback with the original event, giving you additional insight into issues. In order to do this, Sentry needs the `eventId`. For example, to get the `eventId`, you can use <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> or the return value of the method capturing an event.

<PlatformContent includePath="user-feedback/sdk-api-example/" />

Alternatively, you can use the [User Feedback API endpoint](/api/projects/submit-user-feedback/) directly.
```

**Review by lizokm — APPROVED** [review]
I know this is common content, but I have a few wording suggestions. If it sounds good to you, I'll update throughout.

**Comment by tustanivsky** [comment]
> I know this is common content, but I have a few wording suggestions. If it sounds good to you, I'll update throughout.

Those are good points, thanks! I've updated the PR accordingly.

**Comment by supervacuus** [comment]
Please, let's hold back the addition to the docs until sentry-native 0.7.1 is out.

**Comment by supervacuus** [comment]
[0.7.1](https://github.com/getsentry/sentry-native/releases/tag/0.7.1) is out.

**Pull request merged**